cellpack/autopack/Environment.py

```diff
@@ -1579,21 +1579,21 @@ def getSortedActiveIngredients(self, allIngredients):
         priorities1 = []
         ingr2 = []  # priority = 0 or none and will be assigned based on complexity
         priorities2 = []
-        ingr0 = []  # negative values will pack first in order of abs[packing_priority]
+        ingr0 = []  # negative values will pack first in order of abs[priority]
```

Comment:
only change in this file is packing_priority -> priority

Reply:
thank you for the example, I will keep in mind to leave necessary comments to reviewers in large PRs.